easybuild/easyconfigs/i/IMB/IMB-4.1-foss-2016a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'MakeCp'
+
+name = 'IMB'
+version = '4.1'
+
+homepage = 'https://software.intel.com/en-us/articles/intel-mpi-benchmarks'
+description = """
+The Intel® MPI Benchmarks perform a set of MPI performance measurements for point-to-point and global communication operations for a range of message sizes.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = ['%(name)s_%(version)s.tgz']
+source_urls = ['https://software.intel.com/sites/default/files/managed/a3/b2/']
+
+prebuildopts = 'export MPI_HOME=$EBROOTOPENMPI && cd src &&'
+buildopts = '-f make_mpich'
+
+patches = ['IMB-make_mpich.patch']
```

Comment:
missing patch file?

also, please make it clear for which IMB version this patch was created, i.e. rename it to `IMB-4.1_make_mpich.patch` or so...